examples/sdk-usage/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id 'java'
+    id 'com.github.sherter.google-java-format' version '0.8'
+    id "com.jfrog.artifactory" version "4.14.1"
+}
+
+group 'io.opentelemetry'
+version '0.3.0-SNAPSHOT'
```

Comment:
Btw, lets fill an issue to make the examples inherit the main project `version`, or at least a proper way to have them synchronized (the jaeger example still has 0.2.0 version, for example).

Reply:
We already had a discussion and the general opinion was to do not include them in the main build. I will take care of this in #1041